templates/dashboard/order/pdf/invoice.html

```diff
@@ -113,25 +113,37 @@
         {% trans "Subtotal" context "Order subtotal" %}
       </td>
       <td class="right-align">
-        {% gross group.order.get_subtotal_without_voucher html=True %}
+        {% gross group.get_subtotal_without_voucher html=True %}
       </td>
     </tr>
     <tr>
       <td>
         {% trans "Total Shipping Price" context "Order totals header" %}
       </td>
       <td class="right-align">
-        {% gross group.order.get_total_shipping html=True %}
+        {% gross group.shipping_price html=True %}
       </td>
     </tr>
+    {% if first_delivery_group %}
+    {% if group.order.voucher %}
+      <tr>
+        <td>
+          {% trans "Voucher" context "Order voucher header" %} {{ group.order.voucher }}
```

Comment:
Wouldn't it be better if we display voucher like 'Voucher (Big Autumn Discount)'?